components/style/error_reporting.rs

```diff
@@ -27,10 +28,14 @@ pub trait ParseErrorReporter {
 /// TODO(emilio): The name of this reporter is a lie, and should be renamed!
 pub struct StdoutErrorReporter;
 impl ParseErrorReporter for StdoutErrorReporter {
-    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str) {
+    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str, servo_url : Option<&ServoUrl>) {
          if log_enabled!(log::LogLevel::Info) {
              let location = input.source_location(position);
-             info!("{}:{} {}", location.line, location.column, message)
+             match  servo_url {
```

Comment:
@Wafflespeanut @jdm @cbrewster 
does **u.as_str()** not break  when u is None  ?
I avoided this problem Using **match** and **Some(url)** to provide a valid url for
**url.as_str()**